examples/WhitneyMusicBox.html

```diff
@@ -102,7 +102,7 @@
 		onsuccess: function() {
 			var canvas = document.getElementById('mycanvas');
 			canvas.style.width = gw + "px";
-			canvas.style.width = gw + "px";
+			canvas.style.height = gh + "px";
```

Comment:
App worked appropriately on most browsers thanks to "canvas.height = gh;" on line 107 anyways.